src/main/java/com/google/graphgeckos/dashboard/api/DashboardController.java

```diff
@@ -0,0 +1,45 @@
+package com.google.graphgeckos.dashboard.api;
+
+import com.google.graphgeckos.dashboard.datatypes.BuildInfo;
+import com.google.graphgeckos.dashboard.storage.DatastoreRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class DashboardController {
+
+  /**
+   * Provides access to information in the storage.
+   */
+  @Autowired
+  private DatastoreRepository DATA_REPOSITORY;
+
+  /**
+   * Sets response status to Https.BAD_REQUEST (400) if the IllegalArgumentException is thrown
+   * inside the getBuildInfo method.
+   * @param e Exception thrown by {@code DATA_REPOSITORY}.
+   *          {@link DatastoreRepository::getBuildInfo} to learn what causes it.
+   */
+  @ExceptionHandler
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  void onIllegalArgumentException(IllegalArgumentException e) {}
+
+  /**
+   * Handles GET requests from the frontend part of the application.
+   * Gets information from the database via {@code DATA_REPOSITORY}.
```

Comment:
Please leave [one blank line](https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs) between annotations and description

Reply:
Done!